Extensions/xkit_preferences.js

```diff
@@ -1006,7 +1006,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
 
-		var blacklisted_extensions = ["xkit_installer"];
+		var blacklisted_extensions = ["xkit_installer","old_stats"];
```

Comment:
Why would we need to tell? Once this is out, Old Stats is immediately removed and replaced with Old Sidebar anyway since it's 1:1 functionality. At that point, downloading Old Stats makes no sense as it would automatically remove itself and install Old Sidebar.